### PR TITLE
concurrency docs

### PR DESCRIPTION
**Pull request by prha**
## Summary & Motivation
Have something to point users towards in terms of limiting global op concurrency.

## How I Tested These Changes
BK, inspection

### CODE REVIEW

File: docs/content/guides/limiting-concurrency-in-data-pipelines.mdx [hunk 1]
```diff
@@ -387,6 +388,25 @@ height={1638}
...
+
+For example, to globally limit the number of running ops touching redshift to 2, the op must be first tagged with the global concurrency key:
+
+```python file=/concepts/ops_jobs_graphs/job_execution.py startafter=start_global_concurrency endbefore=end_global_concurrency
+@op(tags={"dagster/concurrency_key": "redshift"})
+def my_redshift_op():
+    ...
+```
+
+To specify limits on the number of ops/assets tagged with a particular concurrency key, you can either use the `dagster` CLI command or use Dagit:
```
Comment (erinkcochran87) [inline docs/content/guides/limiting-concurrency-in-data-pipelines.mdx:403]:
```suggestion
To specify limits on the number of ops/assets tagged with a particular concurrency key, you can either use the `dagster` CLI command or the Dagster UI:
```

`Dagit` is now `Dagster UI`

File: docs/content/guides/limiting-concurrency-in-data-pipelines.mdx [hunk 2]
```diff
@@ -387,6 +388,25 @@ height={1638}
...
+
+- To specify a global concurrency limit using the CLI, use `dagster instance concurrency set <concurrency_key> <limit>`
```
Comment (erinkcochran87) [inline docs/content/guides/limiting-concurrency-in-data-pipelines.mdx:405]:
````suggestion
- To specify a global concurrency limit using the CLI, use: 

   ```shell
   dagster instance concurrency set <concurrency_key> <limit>
   ```
````

Suggesting we put this in a code block so users can copy it with a click. You may need to run `make mdx-format` again if you accept this guy

Also, is `concurrency_key` just the value, like `redshift`?

Comment (prha) [reply docs/content/guides/limiting-concurrency-in-data-pipelines.mdx:405]: > Also, is `concurrency_key` just the value, like `redshift`?


Yes

**Review by erinkcochran87 — APPROVED** [review]
Thanks so much, Phil!

**Pull request merged**